### DISCUSSION TITLE
請益 - 請問如何在config.json 可用的模型列表中新增"claude-3-sonnet"與"claude-3-opus"模型?

### DISCUSSION THREAD

**Discussion by tomkuo1124** [GaiZhenbiao/ChuanhuChatGPT#1067] [Q&A]
我想在可用模型列表中增加"claude-3-sonnet"與"claude-3-opus"模型, 可是即使我從Claude workbench找到model name, 加上去還是跑不起來, 會出現錯誤, 請問版主大大以及其他前輩, 我需要從哪裡找到正確的model name?
感恩。

![image](https://github.com/GaiZhenbiao/ChuanhuChatGPT/assets/137477635/8fdf8f92-7709-4342-921e-a4214490b8a2)
![image](https://github.com/GaiZhenbiao/ChuanhuChatGPT/assets/137477635/d15ef726-b356-4bba-9434-b4575b868a36)

**Comment by Keldos-Li** [comment]
这边已经适配，更新最新版的项目主线代码即可～不需要另行在config再添加模型

**Reply by tomkuo1124** [reply]
好的, 非常感謝前輩的回覆, have a nice day. 😊
